Description: init_process_flags.is_lock_free check fails on mipsel
Forwarded: https://github.com/nodejs/node/issues/45152
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2022-10-25

--- a/src/node.cc
+++ b/src/node.cc
@@ -595,7 +595,7 @@
 #endif  // __POSIX__
 }
 
-static std::atomic<uint64_t> init_process_flags = 0;
+static std::atomic<uint32_t> init_process_flags = 0;
 
 static void PlatformInit(ProcessInitializationFlags::Flags flags) {
   // init_process_flags is accessed in ResetStdio(),
